saliency_metrics/metrics/sanity_check/sanity_check.py

```diff
@@ -0,0 +1,72 @@
+from copy import deepcopy
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import torch
+from skimage.metrics import structural_similarity
+from torch import Tensor
+
+from saliency_metrics.metrics.build_metric import METRICS, ReInferenceMetric
+from saliency_metrics.metrics.sanity_check.sanity_check_result import SanityCheckResult
+from saliency_metrics.metrics.serializable_result import SerializableResult
+from saliency_metrics.models.model_utils import freeze_module
+
+
+class SanityCheck(ReInferenceMetric):
+    def __init__(self, ori_model: torch.nn.Module, perturb_layers: List[str], summarized: bool = True) -> None:
```

Comment:
Please stick to the API as I suggested in the GitHub Issue. `ori_model` should be `classifier`.